dbsdk2/config.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+module.exports = {
+    getDbCredentials : (dbName) => {
```

Comment:
change to dbCredentials as an object, not a function

Reply:
It is to provision the usage of more than one databases on the fly